hist/hist/inc/TF1.h

```diff
@@ -258,22 +257,23 @@ class TF1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
    std::vector<Double_t>    fGamma;      //!Array gamma.
    TObject     *fParent{nullptr};     //!Parent object hooking this function (if one)
    TH1         *fHistogram{nullptr};  //!Pointer to histogram used for visualisation
-   TMethodCall *fMethodCall{nullptr}; //!Pointer to MethodCall in case of interpreted function
+   std::unique_ptr<TMethodCall> fMethodCall; //!Pointer to MethodCall in case of interpreted function
    Bool_t      fNormalized{false};    //Normalization option (false by default)
    Double_t    fNormIntegral{};        //Integral of the function before being normalized
-   TF1FunctorPointer  *fFunctor{nullptr}; //! Functor object to wrap any C++ callable object
-   TFormula    *fFormula{nullptr};        //Pointer to TFormula in case when user define formula
-   TF1Parameters *fParams{nullptr};   //Pointer to Function parameters object (exists only for not-formula functions)
+   std::unique_ptr<TF1FunctorPointer>  fFunctor; //! Functor object to wrap any C++ callable object 
+   std::unique_ptr<TFormula>   fFormula;        //Pointer to TFormula in case when user define formula 
+   std::unique_ptr<TF1Parameters> fParams;   //Pointer to Function parameters object (exists only for not-formula functions)
    std::unique_ptr<TF1AbsComposition> fComposition; //! Pointer to composition (NSUM or CONV)
```

Comment:
This should not be a transient member!! 

Please remove the `!` and change from 
```
std::unique_ptr<TF1AbsComposition> fComposition; //! Pointer to composition (NSUM or CONV)
```
to 
```
std::unique_ptr<TF1AbsComposition> fComposition; // Pointer to composition (NSUM or CONV)
```

Reply:
I think this caused the failare observed in the TF1 test. 
Thank you for updating it. I will run then the builds again afterwards